charts/ceph-csi-cephfs/templates/provisioner-deployment.yaml

```diff
@@ -11,6 +11,11 @@ metadata:
     heritage: {{ .Release.Service }}
 spec:
   replicas: {{ .Values.provisioner.replicaCount }}
+  strategy:
+    type: {{ .Values.provisioner.strategy.type }}
+    rollingUpdate:
```

Comment:
this should only be included when `Values.provisioner.strategy.type == RollingUpdate`

Reply:
Yes, need an if check here to check the `strategy.type == RollingUpdate`

Reply:
Thanks for the suggestion! Added the check.